sdcm/cluster.py

```diff
@@ -2979,10 +2979,15 @@ def run_fullscan(self, ks_cf, loader_node, db_node_ip):
         if 'random' in ks_cf.lower():
             ks_cf = random.choice(ks_cf_list)
 
+        FullScanEvent(type='start', loader_node=loader_node, db_node_ip=db_node_ip, ks_cf=ks_cf)
+
         self.log.info('Fullscan for ks.cf: {}'.format(ks_cf))
         cmd = 'select count(*) from {}'.format(ks_cf)
         result = loader_node.remoter.run('cqlsh --no-color --execute "{}" {}'.format(cmd, db_node_ip), verbose=False)
 
+        FullScanEvent(type='finish', loader_node=loader_node,
+                      db_node_ip=db_node_ip, ks_cf=ks_cf,
+                      result={'stdout': result.stdout, 'stderr': result.stderr})
```

Comment:
follow KISS, why did you create new dict, instead of forwarding result object?

Reply:
if pass result object, there is thrown exception from pickle module, that it could not serialise lock object inside the Result Fabric object.

Reply:
hmmm, ok, any way you need to pass exit code too

Reply:
done